src/app/firedancer-dev/commands/rocksdb.c

```diff
@@ -0,0 +1,462 @@
+/* The rocksdb command spawns a smaller topology for replaying shreds from
```

Comment:
`backtest` instead of `rocksdb`

`rocksdb` is just one possible source. we may want to replay out of our blockstore in the future, for example